pgactivity/cli.py

```diff
@@ -356,10 +398,10 @@ def main() -> None:
                 newdataobj = dataobj.try_reconnect()
                 if newdataobj is not None:
                     dataobj = newdataobj
-                    print(term.clear + term.home, end="")
                     break
         except KeyboardInterrupt:
             sys.exit(1)
         else:
-            print(term.clear + term.home, end="")
             break
+        finally:
+            print(memory_stream.getvalue())
```

Comment:
Printing to stdout is intended? or should we use stderr?

Reply:
It was an oversight. Fixed.